atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleTableNameGetter.java

```diff
@@ -33,10 +33,10 @@ String getInternalShortOverflowTableName(ConnectionSupplier connectionSupplier,
             throws TableMappingNotFoundException;
 
     Set<TableReference> getTableReferencesFromShortTableNames(
-            ConnectionSupplier connectionSupplier, Set<String> shortTableNames) throws TableMappingNotFoundException;
+            ConnectionSupplier connectionSupplier, Set<String> shortTableNames);
```

Comment:
We now return partial results if some of the tables correspond to unmapped tables instead of throwing